src/trace/listener.ts

```diff
@@ -111,7 +114,9 @@ export class TraceListener {
       options.childOf = spanContext;
     }
     options.type = "serverless";
-    options.resource = this.handlerName;
+    options.service = "aws.lambda";
+    options.resource = this.context && this.context.functionName;
```

Comment:
In the case that `this.context` is non-existent/falsy, would `resource` display the falsy value as a string?

Reply:
What should the `resource` be if the `context` is `undefined`? Currently, `context` gets initialized here: https://github.com/DataDog/datadog-lambda-js/blob/b64a50e3316fff7df9d0d17ae56db06180690262/src/trace/listener.ts#L61
After the initialization, if `context` is `undefined`, then resource will get set to `undefined` - maybe we shouldn't set it at all?

Reply:
Yeah I think that's a good idea - if `context` doesn't exist then let's skip assigning a value to `resource` altogether. Seems like we currently do that in our ruby and python layers.

Reply:
Fixed.